jobserver/views/status.py

```diff
@@ -43,6 +44,6 @@ def get_stats(backend):
                 "show_warning": show_warning(last_seen),
             }
 
-        backends = Backend.objects.all()
+        backends = Backend.objects.order_by(Lower("name"))
```

Comment:
Makes sense. We are applying specific logic via model functions here I think rather than generic ptyhon

Reply:
We'd have to sort the QuerySet in Python without this too, something like `sorted(backends, key=lambda b: b.name.lower())`.  That would convert it to a Python iterable so we'd lose any QuerySet functionality we might want to use.  We don't _need_ that here but always useful to be able to do something simple like this without ejecting from a QuerySet.